vendors/ti/boards/cc3220_launchpad/aws_demos/config_files/ota_config.h

```diff
@@ -43,6 +66,11 @@
  */
 #define otaconfigLOG2_FILE_BLOCK_SIZE           10UL
 
+/**
```

Comment:
The comment is for the number of buffers, please keep it number of blocks requested plus 1